app/views/tv_series_viewings/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<% content_for(:title, "TV Series Viewings") %>
```

Comment:
what does this do again? i forget 😅

Reply:
I _think_ this makes the browser tab show the title for is. 😅